### PR TITLE
Updated duplicati (1.3.4)

### PR DESCRIPTION
**Pull request by tsparber**
Tick #17075

### CODE REVIEW

File: Casks/duplicati.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@
   version '1.3.4'
   sha256 'd97065d012fb929fa4e37b5efab2d246b6aef667f892a95301a1271f704963f2'
 
-  # googlecode.com is the official download host per the vendor homepage
-  url "https://duplicati.googlecode.com/files/Duplicati%20#{version}.dmg"
+  # storage.googleapis.com is the official download host per the vendor homepage
+  url "https://storage.googleapis.com/google-code-archive-downloads/v1/code.google.com/duplicati/Duplicati%20#{version}.dmg"
```
Comment (tsparber) [inline Casks/duplicati.rb]: I'm no sure about the comment.

**Comment by tsparber** [comment]
Google maybe has a problem with the spaces. Waiting a bit on https://github.com/duplicati/duplicati/issues/1545#issuecomment-172825070 or will suggest deletion of the cask otherwise.

File: Casks/duplicati.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/duplicati.rb]: Think of the rule this way: if the link provided on the homepage redirects you to a new domain, it’s still OK to have the comment.

I do know there are cases where the comment can be a bit weirder, so I am thinking of a new substitute comment that’ll make everything clearer and be possible to use in more cases.

File: Casks/duplicati.rb [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
   name 'Duplicati'
-  homepage 'http://www.duplicati.com/'
+  homepage 'http://www.duplicati.com'
```
Comment (vitorgalvao) [inline Casks/duplicati.rb]: We actually prefer the trailing `/`, as it’s the proper way to present a URL.

**Comment by tsparber** [comment]
Closing in favor of #17338

**Pull request closed without merging**